pages/samvera/glossary/hyrax_2.1/glossary.md

```diff
@@ -0,0 +1,100 @@
+
+## Mediated deposit
+The group of steps necessary for reviewing a deposited work, culminating in approval before publishing it for public viewing. Commonly used in the context of collecting Electronic Theses and Dissertations or staff-facing metadata workflows for digital collections.
+
+## Participants
+Participants for Administrative Sets allow the repository manager to define what individual users or groups of users can do with or to items in an Admin Set, such as deposit, approve, edit, or withdraw.
+
+Participants for Collection Types allow the repository manager to define which individual users or groups of users can have roles related to collections of a type.  A manager is given full edit access to collections of this type.  A creator is allowed to create new collections of this type.  ([more information...](collection-type-participants.html))
+
+See also [Sharing](#sharing) which describes roles for Collections and Works.
+
+**Permission Templates**
+A template that defines the Visibility, Release, Workflow, and Participants for an Admin Set or Collection.  The template is applied to works when the work is created.  The template is created through the UI when an Admin Set or Collection is created or edited.  Permission Templates are applied to works when the item is created.  Changes to the template are not applied to existing works and will only effect new works created after the change.
+
+**Collection Type Templates**
+A template that defines the Participants for Collection based on it's Collection Type.  The template is applied to collections when the collection is created.  The template is created through the UI when a Collection Type is created or edited.  Collection Type Templates are applied to collections when the item is created.  Changes to the template are not applied to existing collections and will only effect new collections created after the change.
+
+## Mediated Deposit Workflow
+The group of steps necessary for reviewing an ingested item and its metadata, culminating in approval before publishing it for public viewing.  ([more information...](https://github.com/samvera/sufia/wiki/Mediated-Deposit-Workflow))
+ 
+## Proxy
+A user who can deposit works on behalf of another user. Example. Research Assistant deposits articles written by a Professor.
+
+## Resource Type
+A classification for content that is assigned to the Work by the depositor. Resource Types are discoverable via facet by default. Examples include: Article, Dataset, Image.
+
+## Roles
+
+**System Roles**
+
+Repository Administrator is the only system-wide role that exists out of the box. Repository Administrators:
+- Can see, edit, delete all works, collections, admin sets regardless of access controls and workflows on the objects.
+- Only Repository Administrators can
+  - create Administrative Sets
```

Comment:
@elrayle will this be true with CE?

Reply:
Users will be able to give other users permission to create Admin Sets without the other users having to be admin users.